napari/layers/_layer_actions.py

```diff
@@ -19,6 +20,7 @@
 import numpy as np
 from typing_extensions import TypedDict
 
+from .._qt.widgets.qt_axis_popup import QAxisPopup
```

Comment:
We've tried to keep this file free on Qt imports so we can consider on the headless side